ddtrace/internal/telemetry/__init__.py

```diff
@@ -3,9 +3,16 @@
 This is normally started automatically when ``ddtrace`` is imported. It can be disabled by setting
 ``DD_INSTRUMENTATION_TELEMETRY_ENABLED`` variable to ``False``.
 """
-from .writer import TelemetryWriter
+import os
```

Comment:
Can we not import the `telemetry_writer` here at all? This is all internal so we wouldn't break anything and ideally we want `__init__.py` files to be empty. cc @mabdinur